[Docs] Update model monitoring docs (#6213)
mlrun/mlrun: docs/model-monitoring/index.md

<img src="../_static/images/mm_metrics.png" width="700" >

## Selecting the streaming and TSDB platforms

> Model monitoring supports open-source streaming platforms such as Kafka, and MySQL (8.0 and higher), 
> in addition to integration with the Iguazio AI platform V3IO data layer. It supports TDEngine, and V3IO for the TSDB platform.

Reviewer: ```suggestion
Model monitoring supports Kafka or V3io as streaming platforms, and TDEngine or V3IO as TSDB platform.
In addition, internal model-monitoring metadata can be saved in MySQL (8.0 and higher) or V3IO.  
```

Before you deploy the model monitoring or serving function, you need to {py:meth}`set the credentials <mlrun.projects.MlrunProject.set_model_monitoring_credentials>`. 
There are three credentials you can set, and each one can have a different value. For example:
```

> stream_path = kafka://<some_kafka_broker>:<port>
> tsdb_connection = taosws://<username>:<password>@<host>:<port>

Reviewer: ```suggestion
stream_path = kafka://<some_kafka_broker>:<port>   | "v3io"
tsdb_connection = taosws://<username>:<password>@<host>:<port>.  | "v3io"
```
also needs to document endpoint_store_connection.

```
or
```
stream_path = v3io
tsdb_connection = taosws://<username>:<password>@<host>:<port>

> ```

Reviewer: remove
